client/verta/verta/endpoint/build/_build.py

```diff
@@ -59,13 +60,11 @@ def _get(cls, conn: _utils.Connection, workspace: str, id: int) -> "Build":
         response = _utils.make_request("GET", url, conn)
         _utils.raise_for_http_error(response)
 
-        return cls(conn, response.json())
+        return cls(conn, workspace, response.json())
 
     @classmethod
     def _list_model_version_builds(
-        cls,
-        conn: _utils.Connection,
-        id: int,
+        cls, conn: _utils.Connection, id: int, workspace: str
```

Comment:
I'd prefer if we had `workspace` come right after `conn` for consistency with all our of other functions (plus it's an internal and yet-unreleased method, so it's fine), but not a huge deal.

Reply:
Easy fix.  Done